### PR TITLE
initial thoughts of target selection

### LINKED ISSUE #6: Determine a candidate list of projects for the initial run
There was some discussion on this during the last meeting. 

It was mentioned that we need a candidate list of projects which are:
1. Active and "healthy"
2. High number of users

Once we get the the candidate list, we can pick one most suited for differential analysis and begin.

Projects we worked with so far: Libtiff, Nginx, HTTPD, OpenSSL, Libav, FFMpeg

### PR DESCRIPTION
**Pull request by goern**
initial ideas of criteria to down select a list of project to target

closes #6 

Signed-off-by: Christoph Görn <goern@redhat.com>

@xtuchyna

### CODE REVIEW

File: target_projects.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Initial Target Project List
+
+We want to focus our effort on a small list of projects, that we think we can help to a high degree.
+
+### Selection Criteria
+
+* focus on Fedora SIG with a lot of packages, as this might indicate a relative importance to Fedora
+* just C or Python language projects
+* relevance of the SIG to Red Hat Enterprise Linux or OpenShift
+* projects should be medium active, as this might indicate that we can focuse spares resources, and show our value
```
Comment (xtuchyna) [inline target_projects.md:10]: I would say a project should be active (medium/high) and indicate somehow a lowered health in means of maintainance - that means increased number of security related meta-information indicators

**Comment by goern** [comment]
let's start traversing https://src.fedoraproject.org/groups

**Comment by xtuchyna** [comment]
> let's start traversing https://src.fedoraproject.org/groups

You mean doing the data extraction for all the upstream repos listed there?

**Comment by goern** [comment]
> You mean doing the data extraction for all the upstream repos listed there?

ja. might be a nice dashboard. if we can get it done in a few days.

**Comment by goern** [comment]
@xtuchyna could you please start aggregating MI data of all the source repos, please create a YAML file in this repo containing all the source repositories.

**Comment by goern** [comment]
seeAlso https://github.com/thoth-station/mi/issues/403

**Pull request closed without merging**